generators/builtins/ranges.kt

```diff
@@ -16,9 +16,11 @@
 
 package org.jetbrains.kotlin.generators.builtins.ranges
 
-import org.jetbrains.kotlin.generators.builtins.*
-import org.jetbrains.kotlin.generators.builtins.generateBuiltIns.*
+import org.jetbrains.kotlin.generators.builtins.ProgressionKind
 import org.jetbrains.kotlin.generators.builtins.ProgressionKind.*
+import org.jetbrains.kotlin.generators.builtins.areEqualNumbers
+import org.jetbrains.kotlin.generators.builtins.generateBuiltIns.BuiltInsSourceGenerator
+import org.jetbrains.kotlin.generators.builtins.hashLong
```

Comment:
Please retain star imports here.

Reply:
Those same changes have been added [in 2022](https://github.com/JetBrains/kotlin/commit/d788a927c4dba94be67effdfd2458e9d4ff2e4f8#diff-eb28dacf319e70cecbddb982556f4d089830fa24f71415fd4a8563c46bdfc1f4]